docker-compose.yml

```
@@ -1,9 +1,7 @@
 version: "3.8"
 services:
   inhouse_bot:
-    build:
-      context: .
-      dockerfile: ./Dockerfile
+    image: ghcr.io/adamschachne/inhouse_bot:latest
```

Remove PII

```suggestion
    image: inhouse_bot:latest
```

You need to `docker pull ghcr.io/adamschachne/inhouse_bot:latest` then `docker tag ghcr.io/adamschachne/inhouse_bot:latest inhouse_bot:latest` OR `docker compose build`.